### PR TITLE
OCD-4213: Subscriptions

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: chpl/chpl-resources/src/main/resources/email.properties [hunk 1]
```diff
@@ -357,3 +357,22 @@ developer.missingAttestationChangeRequest.report.paragraph=Emails were sent to t
...
+subscriber.confirmMessage.paragraph1=You are almost ready to receive notifications from the CHPL! \
+Please <a href="%s">click here to confirm your email address</a> and begin receiving notifications.
+subscriber.confirmMessage.paragraph2=If you no longer wish to receive notifications from the CHPL \
+<a href="%s">click here to unsubscribe</a>.
+subscriber.confirm.url=/#/subscriptions/confirm/%s
+subscriptions.manage.url=/#/subscriptions/manage/%s
+
+#Subscription Observation Emails
+observation.notification.subject=CHPL Subscription Notification
+observation.notification.introduction=Below are items in the CHPL with changes to which you have subscribed:
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/email.properties]:
```suggestion
observation.notification.introduction=You have subscribed to receive notifications regarding changes to the following CHPL items:
```

File: chpl/chpl-resources/src/main/resources/email/chpl-email-skeleton.html [hunk 2]
```diff
@@ -20,6 +20,11 @@
     border: 1px solid #eeeeee;
     font-family: Arial, Helvetica, sans-serif;
 }
+
+#chpltable caption {
+    caption-side:bottom;
+    text-align:left;
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/email/chpl-email-skeleton.html]:
```suggestion
    caption-side: bottom;
    text-align: left;
```

Standard format for HTML styles (at least in the UI)

File: chpl/chpl-resources/src/main/resources/email.properties [hunk 3]
```diff
@@ -30,7 +30,7 @@ during Log In, or by contacting your local administrator.
 #password changed email
 user.updatedPassword.subject=CHPL Password Changed
 user.updatedPassword.heading=Your Password Has Been Changed
-user.updatedPassword.paragraph1=We successfully processed a request to change your�password on %s.
+user.updatedPassword.paragraph1=We successfully processed a request to change your password on %s.
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/email.properties]: Something looks weird on this line, but I can't do a "suggested code update" to fix it... Do you see the diff here?

**Review by andlar — DISMISSED** [review]

File: chpl/chpl-resources/src/main/resources/email.properties (hunk 3 above)
Comment (kekey1) [reply chpl/chpl-resources/src/main/resources/email.properties]: Wow.. this line is highlighted as a diff for me, but nothing looks odd about it. In the email Github sent me with your comment, however, I saw a non-UTF-8 character.

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/subscriber/validation/SubscriberValidationContext.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    private SubscriberDao subscriberDao;
+    private ErrorMessageUtil errorMessageUtil;
+
+    public SubscriberValidationContext(UUID subscriberId,
+            SubscriberDao subscriberDao,
+        ErrorMessageUtil errorMessageUtil) {
+        this.subscriberId = subscriberId;
+        this.subscriberDao = subscriberDao;
+        this.errorMessageUtil = errorMessageUtil;
+    }
```
Comment (tmy1313) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/subscriber/validation/SubscriberValidationContext.java]: Could this be replaced with `@AllArgConstructor`?

**Review by tmy1313 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**